libraries/AP_Math/crc.cpp

```diff
@@ -347,3 +347,16 @@ uint8_t crc_sum8(const uint8_t *p, uint8_t len)
     sum = 0xff - ((sum & 0xff) + (sum >> 8));
     return sum;
 }
+
+/// 8 bit CRC used by the BLHeli and FETtec ESC telemetry protocols
+uint8_t crc8_telem(uint8_t crc, uint8_t crc_seed)
```

Comment:
I think this is the same as crc8_dvb(crc, crc_seed, 0x7) - we should do that instead, might not even need a separate function